README.md

```diff
@@ -0,0 +1,42 @@
+# cross_modality_query
+
+Server-side code for HuBMAP Cells API.
+[Python](https://github.com/hubmapconsortium/hubmap-api-py-client) and 
+[Javascript](https://github.com/hubmapconsortium/hubmap-api-js-client) clients available.
+
+## Usage
+
+This is only a high-level outline of the API:
+See the [Python client examples](https://github.com/hubmapconsortium/hubmap-api-py-client/tree/main/examples) for details.
+
+The API supports a small number of basic operations which can be combined to construct more complex queries.
+The API runs at a **`base_url`**: Currently `https://cells.dev.hubmapconsortium.org/api` is available.
+
+Issuing a `POST` to `{base_url}/{output_type}/` (where `output_type` is `cell`, `organ`, `gene`, or `cluster`),
```

Comment:
"output_type" is not particularly informative. Are those the entities that will be retrieved by the query?

Reply:
Yes, the output_type is the type of entity in the set that is constructed in response to the query.